autonomy/src/controllers.py

```diff
@@ -28,45 +27,52 @@ class ProportionalController:
 
     @dataclass(frozen=True)
     class Constants:
-        DEPTH_SPEED: int = 1
-        ROTATION_SPEED: int = 1
-        LINEAR_SPEED: int = 1
-        FRONT_TORPEDO_SPEED: int = 5
-        BACK_TORPEDO_SPEED: int = 4
+        # PWM settings for thruster control
+        PWM_NEUTRAL: int = 1500       # Neutral position
+        PWM_MIN: int = 1300           # Max reverse
+        PWM_MAX: int = 1700           # Max forward
+
+        DEPTH_PWM_ADJUST: int = 50       # PWM adjustment for depth
+        ROTATION_PWM_ADJUST: int = 50    # PWM adjustment for rotation
+        LINEAR_PWM_ADJUST: int = 50      # PWM adjustment for linear movement
+        
         DELTA: float = 0.01
-        SPEED_TRANSLATION: dict = field(default_factory=lambda: {
-            1: 1550,
-            2: 1600,
-            3: 1650,
-            4: 1700,
-            5: 1300,
-            6: 1400,
-            7: 1350
-        })
+        
         TOTAL_THRUSTERS: int = 8
```

Comment:
This should be 4, also depth and torpedo motors are not controlled by their thruster ID topics

Reply:
Makes sense, I suppose it can stay as is then